components/MovieCard/index.js

```diff
@@ -16,28 +16,32 @@ export default function MovieCard() {
       });
   }, []);
 
+  const handleClick = () => {
+    setToggle(!toggle);
+  };
+
   return (
     <>
       <MovieList>
         {movies.map((movie, index) => (
-          <MovieListItem
-            key={movie.id}
-            onClick={() => {
-              setActiveItem(index);
-            }}
-          >
-            <CardImage src={API_IMG + movie.poster_path}></CardImage>
-            {index === activeItem && (
-              <>
+          <MovieListItem key={movie.id}>
+            <CardImage
+              onClick={handleClick}
+              src={API_IMG + movie.poster_path}
+            ></CardImage>
+            <>
+              {toggle ? (
                 <CardBody>
                   <UserRating>User-Rating: {movie.vote_average}</UserRating>
                   <StyledButton>+</StyledButton>
                   <MovieTitleH3>{movie.original_title}</MovieTitleH3>
                   <ReleaseDate>Release-Date: {movie.release_date}</ReleaseDate>
                   <Overview>Overview: {movie.overview}</Overview>
                 </CardBody>
-              </>
-            )}
+              ) : (
```

Comment:
Very nice ternary, but do you really need an else statement? Maybe if you write: toggle &&  and than you can leave out the else statement. Otherwise good job!